maksze/telemetry-bundle/0.4/config/packages/maksze_telemetry.yaml

```diff
@@ -0,0 +1,14 @@
+# Configuration file for the MaksZeTelemetry bundle
+#
+# For more information about the bundle settings visit:
+# https://github.com/maksze/MaksZeTelemetryBundle/blob/master/README.md
+
+maksze_telemetry:
+    yandex_metrika:
+        #- {id: '%env(YANDEX_METRIKA_ID)%'}
+    yandex_webmaster:
+        #- {id: '%env(YANDEX_WEBMASTER_ID)%'}
+    google_search_console:
+        #- {id: '%env(GOOGLE_SEARCH_CONSOLE_ID)%'}
```

Comment:
No need to use comments here, right?

I would assume a user have at least two of these values ready when installing the bundle.

Reply:
I do not agree, in Russia we often use Yandex services than Google